### PR TITLE
Feature/1 initial content

### LINKED ISSUE #1: Create initial content for the dogu

### PR DESCRIPTION
**Pull request by Subaro**
Resolves #1

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+void testStaticContentAccess(K3d k3d) {
+    k3d.waitForDeploymentRollout("nginx-ingress", 300, 5)
+
+    // determine free port with python
+    String port = sh(script: 'echo -n $(python3 -c \'import socket; s=socket.socket(); s.bind(("", 0)); print(s.getsockname()[1]); s.close()\');', returnStdout: true)
+
+    // the process is automatically terminated when canceling/terminating the build
+    k3d.kubectl("port-forward service/nginx-ingress ${port}:443 &")
+
+    sh "sleep 5"
```
Comment (nhinze23) [inline Jenkinsfile:132]: It is possibile to wait here for the port-forward with a timeout?

File: bats.mk [hunk 2]
```diff
@@ -0,0 +1,62 @@
+WORKSPACE=/workspace
```
Comment (nhinze23) [inline bats.mk:1]: This file could be moved to our Makefiles.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+MAKEFILES_VERSION=6.0.3
```
Comment (nhinze23) [inline Makefile:1]: Use 6.1.0

File: dogu.json [hunk 4]
```diff
@@ -0,0 +1,81 @@
+{
+  "Name": "k8s/nginx-static",
+  "Version": "1.23.1-0",
+  "DisplayName": "Nginx Static",
+  "Description": "Nginx Static is a webserver responsible to provide static content in the Cloudogu EcoSystem.",
```
Comment (nhinze23) [inline dogu.json:5]: You could mention that this is a webserver for multinode.

File: nginx-build/build.sh [hunk 5]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (nhinze23) [inline nginx-build/build.sh:1]: Why do you need this script? Local dev? Maybe the Makefile is a better place for this?

File: .env.template [hunk 6]
```diff
@@ -0,0 +1,2 @@
+export K8S_CLUSTER_ROOT=<your_cluster_root_path>
+export POD_NAMESPACE=$(shell kubectl config view --minify -o jsonpath='{..namespace}')
```
Comment (nhinze23) [inline .env.template:2]: Is POD_NAMESPACE used?

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+# nginx-static Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+###
```
[the remark this answers is not in the document]
Comment (Subaro) [reply CHANGELOG.md:10]: Done

[the remark this answers is not in the document]
Comment (Subaro) [reply CHANGELOG.md:10]: Done

File: Dockerfile [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+    && /build.sh \
+    && rm -rf /var/cache/apk/* /build
+
+
+FROM registry.cloudogu.com/official/base:3.15.3-1
+LABEL maintainer="hello@cloudogu.com" \
+      NAME="nginx-static" \
+      VERSION="0.0.0"
+
+ENV WARP_MENU_VERSION=1.5.0 \
```
[the remark this answers is not in the document]
Comment (Subaro) [reply Dockerfile:29]: Done

File: Jenkinsfile (hunk 1 above)
Comment (Subaro) [reply Jenkinsfile:132]: The execution of line 130 automatically waits for the port forwarding. I do not know anymore why the sleep is needed at all.

I will try to remove the sleep.

File: Makefile (hunk 3 above)
Comment (Subaro) [reply Makefile:1]: Updated to 6.2.0. But we need another makefiles bugfix

File: bats.mk (hunk 2 above)
Comment (Subaro) [reply bats.mk:1]: Sure. Do you want to do this with the other changes?

File: dogu.json (hunk 4 above)
Comment (Subaro) [reply dogu.json:5]: Done

File: nginx-build/build.sh (hunk 5 above)
Comment (Subaro) [reply nginx-build/build.sh:1]: To build the nginx inside the Dockerfile.

File: .env.template (hunk 6 above)
Comment (Subaro) [reply .env.template:2]: Yes. It is used generally for the makefiles

File: Makefile (hunk 3 above)
Comment (nhinze23) [reply Makefile]: Updated to 6.3.0

**Pull request merged**